molecule/default/destroy.yml

```diff
@@ -0,0 +1,11 @@
+- name: Destroy
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  no_log: |-
+    {{ not (lookup('env', 'MOLECULE_DEBUG') | bool or
+       molecule_yml.provisioner.log|default(false) | bool) }}
+  roles:
+    - name: molecule_openstack_ci
```

Comment:
I love the test-ception going on here!

Reply:
This in particular was the part where I expected everything to come crashing down, but...nope. It just worked on the first shot.